run_configs/um/build_um.py

```diff
@@ -124,14 +124,12 @@ def um_atmos_safe_config(revision, two_stage=False, opt='Og'):
 
             # fparser2 fails to parse this file, but it does compile.
             special_measure_analysis_results=[
-                AnalysedFile(
-                    fpath=Path(
-                        config.build_output / "casim/lookup.f90"),
+                ParserWorkaround(
```

Comment:
I like that thinking, I think we can go even further. When we add the manual results, we could just check if the file was analysed. That would mean the analyser didn't crash for that file, so the workaround is no longer necessary. Sound like a plan?

Reply:
Automation is great. How are we going to communicate to the developer that their workaround isn't necessary any more? It may have to be an exception to force their hand into doing something.

Reply:
Ok, exception it is then. :+1: